tidb-distributed-execution-framework.md

```diff
@@ -0,0 +1,124 @@
+---
+title: TiDB Backend Task Distributed Execution Framework
+summary: Learn the use cases, limitations, usage, and implementation principles of the TiDB backend task distributed execution framework.
+---
+
+# TiDB Backend Task Distributed Execution Framework
+
+> **Warning:**
+>
+> This feature is an experimental feature. It is not recommended to use it in production environments.
+
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, this feature is only applicable to Dedicated Tier clusters. You cannot use it on Serverless Tier clusters.
```

Comment:
```suggestion
> Currently, this feature is only applicable to TiDB cloud Dedicated and on primise clusters . You cannot use it on TiDB cloud Serverless clusters.
```

Reply:
The content wrapped by `<CustomContent platform="tidb-cloud">` will only be seen by TiDB Cloud docs readers, so it's ok not to mention on-premises.